google/cloud/forseti/scanner/scanners/external_project_access_scanner.py

```diff
@@ -291,19 +288,33 @@ def _retrieve(self):
 
         user_emails = get_user_emails(self.service_config)
 
-        for user_email in user_emails:
-            user_count += 1
-            try:
-                user_crm_client = self._get_crm_client(user_email)
-
-                project_ids = extract_project_ids(user_crm_client)
-                ancestries = get_project_ancestries(user_crm_client,
-                                                    project_ids)
+        # for user_email in user_emails:
```

Comment:
temporarily replace this for loop with only one iteration, starting line 305, to shorten each testing cycle.